### PR TITLE
add documentation for --eval-file flag

### PR DESCRIPTION
**Pull request by najumancheril**

### CODE REVIEW

File: tools/repl.md [hunk 1]
```diff
@@ -53,6 +53,51 @@ Uncaught AssertionError
...
+you commonly use in the REPL, or modifying the runtime in some way.
+
+Files can be specified as paths or URLs. URL files are cached and can be
+reloaded via the `--reload` flag.
+
+If `--eval` is also specified, then `--eval-file` files are run before the
+`--eval` code.
+
+```
+$ deno repl --eval-file=https://examples.deno.land/hello-world.ts,https://deno.land/std@0.135.0/encoding/ascii85.ts
```
Comment (bartlomieju) [inline tools/repl.md:69]: Replace hardcoded version of `deno_std` with `@$STD_VERSION` makers, they will be automatically filled with the latest available version

**Review by bartlomieju — APPROVED** [review]

**Review by bartlomieju — CHANGES_REQUESTED** [review]
Ooops, this was meant to be "Request changes"

**Comment by najumancheril** [comment]
Do we need the $STD_VERSION?

My system interprets this as an undefined bash environment variable and evaluates it to "". So I see std@.

`$ ./target/debug/deno repl --reload --eval-file=https://examples.deno.land/hello-world.ts,https://deno.land/std@$STD_VERSION/encoding/ascii85.ts
Download https://examples.deno.land/hello-world.ts
Hello, World!
Download https://deno.land/std@/encoding/ascii85.ts
Warning Implicitly using latest version (0.135.0) for https://deno.land/std/encoding/ascii85.ts
Download https://deno.land/std@0.135.0/encoding/ascii85.ts
Deno 1.20.5
exit using ctrl+d or close()
>`

It shows the same thing if I omit the version entirely. Should I do this in the manual example?

`$ ./target/debug/deno repl --reload --eval-file=https://examples.deno.land/hello-world.ts,https://deno.land/std/encoding/ascii85.ts
Download https://examples.deno.land/hello-world.ts
Hello, World!
Download https://deno.land/std/encoding/ascii85.ts
Warning Implicitly using latest version (0.135.0) for https://deno.land/std/encoding/ascii85.ts
Download https://deno.land/std@0.135.0/encoding/ascii85.ts
Deno 1.20.5
exit using ctrl+d or close()
>`

**Comment by bartlomieju** [comment]
@najumancheril yes, when the site is rendered this marker is replaced with the latest available version of `deno_std` (or a version that matches selected CLI version in manual). So `https://deno.land/std@$STD_VERSION/http/server.ts` gets turned into `https://deno.land/std@0.135.0/http/server.ts` automatically.

**Review by bartlomieju — APPROVED** [review]
LGTM, thank you @najumancheril

**Pull request merged**